### DISCUSSION TITLE
how to solve fps

### DISCUSSION THREAD

**Discussion by vanshunzpt** [naverz/zepeto-studio-global#21] [Q&A]
my fps became drop into 18 and it has 4 million tris , i saw first time zepeto has 3 million tris but i has 60 fps kinda weird

**Comment by jhslee** [comment]
Hello @vanshunzpt! 👋 

hmm, that is odd 🤔 Can you please describe the environment you develop in, as well as the devices you were testing on? Also what software were you using to track the FPS on the device?

**Reply by vanshunzpt** [reply]
i use msi with nvidia 950m i'm trying with zepeto environment its feeling fast 70 fps with 3 million tris
